### ISSUE TITLE
Microsoft.SqlServer.Types x64 DLLs are missing after restore from nuget. 

### ISSUE THREAD

**Issue by Grauenwolf** [FransBouma/RawDataAccessBencher#37]
Under SqlServerTypes, I find the x86 DLLs but not the x64 versions. 

Work-around:

Reinstalling the NuGet package Microsoft.SqlServer.Types, then reverting any changes, seems to restore the missing files.

**Comment by FransBouma** [comment]
Yeah I think it's an issue in their nuget package. Restore doesn't execute their install script and for some reason doesn't restore the dlls. When I remove/reinstall the package nothing in any config file / proj file changes, so my guess is their package installation doesn't work properly.

**Comment by fredericDelaporte** [comment]
There is something else though. [Here in this repository](https://github.com/FransBouma/RawDataAccessBencher/tree/master/NH/DAL/Model/SqlServerTypes), the x86 files are present, saved in git. So why not adding the x64 files too (and in L2S by the way)? Or maybe clean them all and document in readme.md the additional step for having theses references fixed.

**Comment by FransBouma** [comment]
I wasn't aware there were dlls committed to the repo :) To my knowledge they were all loaded from the package folder, but admitted I haven't paid much attention to this particular detail. I simply added the nuget package to make the code compile (the types aren't used in queries, but the database contains the types, so for completeness I kept them in the model). So when I  had to restore the packages I indeed saw the references were broken, at least for some projects. Re-adding them through nuget did work, but it's indeed a sucky experience. 

Adding the dlls is perhaps a better idea, although I don't know if it fixes the restore issue (as it appears it will load dlls from the gac...
